openapi/components/schemas/TransactionLimitAmount.yaml

```
@@ -0,0 +1,15 @@
+type: object
+title: LimitAmount
+nullable: true
+properties:
+  amount:
+    type: number
+    format: double
+    example: 275.35
+    description: The limit amount.
+  currency:
+    $ref: ./CurrencyCode.yaml
+  resetTime:
+    type: string
+    format: date-time
+    description: The time the limit resets which may be used for some user interfaces.
```

suggestion: At first read I was not sure which time was being reset.
I understood on the second or third read. My suggestion here is to help make this more explicit.

```suggestion
    description: The date and time in which the limit amount resets. This value may be used for some user interfaces.
```